roles/yum_repo/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# defaults file for yum-repo
+repo_tag: "generic"
+repo_name: "generic"
+repo_baseurl: "generic"
+repo_enabled: "0"
+repo_gpg_check: "0"
```

Comment:
I believe it's reasonable to assume the repo should be enabled and the gpg checked. So I'd change both of these to "1". Then people can override them when different.